azurerm/data_source_kubernetes_cluster.go

```diff
@@ -77,6 +77,20 @@ func dataSourceArmKubernetesCluster() *schema.Resource {
 								},
 							},
 						},
+
+						"azure_policy": {
+							Type:     schema.TypeList,
+							MaxItems: 1,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"enabled": {
+										Type:     schema.TypeBool,
+										Required: true,
```

Comment:
Also, this should be `Computed: true` instead
```suggestion
										Computed: true,
```

Reply:
Done